Mathlib/LinearAlgebra/Matrix/PosDef.lean

```diff
@@ -315,6 +333,43 @@ theorem transpose {M : Matrix n n R} (hM : M.PosDef) : Mᵀ.PosDef := by
   rw [mulVec_transpose, Matrix.dotProduct_mulVec, star_star, dotProduct_comm]
 #align matrix.pos_def.transpose Matrix.PosDef.transpose
 
+protected theorem diagonal_of_selfAdjoint [DecidableEq n] [NoZeroDivisors R] (d : n → R)
```

Comment:
This is misnamed and contains a `sorry`.

Reply:
Whoops, I meant to unrequest your review when I split this PR, but I guess I failed to do so.